java/src/main/java/com/cybersource/ws/client/SecurityUtil.java

```diff
@@ -247,34 +260,35 @@ public static Document handleMessageCreation(Document signedDoc, String merchant
      * @throws SignException
      */
     public static Document createSignedDoc(Document workingDocument,String keyAlias, String password,Logger logger) throws SignException {
-        
+
         logger.log(Logger.LT_INFO, "Signing request...");
-        //long startTime = System.nanoTime();
-        WSSecHeader secHeader = new WSSecHeader();
+        long startTime = System.nanoTime();
```

Comment:
comment this while promoting. you can have uncomment locally.

Reply:
Updated.